library/entropy.c

```diff
@@ -576,10 +578,12 @@ int mbedtls_entropy_source_self_test( int verbose )
 int mbedtls_entropy_self_test( int verbose )
 {
     int ret = 1;
+#if !defined(MBEDTLS_TEST_NULL_ENTROPY)
```

Comment:
But then if I understand you correctly, you would exclude the line that prints "ENTROPY test". That can be useful if the test hangs halfway through the test. You which test was being executed.